surreal/env/make_env.py

```diff
@@ -12,13 +13,26 @@
     )
 
 
+def make_env_config(env_config, mode=None):
+    """
+    Same as make_env, but explicitly deallocates the environment to avoid 
+    further complications
```

Comment:
Can you change this comment to be more explicit, i.e. describe the segfault issue we had and how multiprocessing is unsafe with openGL context. Let's document this major incident well.